code/workspaces/web-app/src/auth/auth.js

```diff
@@ -90,10 +79,10 @@ function processHash(authResponse) {
 }
 
 function processResponse(authResponse) {
-  const state = processState(authResponse.state);
+  const state = undefined;
   const appRedirect = state ? state.appRedirect : undefined;
-  const expiresAt = authResponse.expiresAt || expiresAtCalculator(authResponse.expiresIn);
-  const identity = authResponse.identity || processIdentity(authResponse.idTokenPayload);
+  const expiresAt = authResponse.expiresAt || expiresAtCalculator(authResponse.expires_at);
```

Comment:
Shouldn't this be expires_in ?

Reply:
Yes that's a good point - I guess you mean 
```const expiresAt = authResponse.expires_at || expiresAtCalculator(authResponse.expires_at);```

Reply:
No, I think the last bit should be expires_in - the expiresAtCalculator takes an expires_in interval and converts it to an expires_at value.